gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
```

Comment:
Please bump in dockerfile too

Reply:
It is already 7.6 for this branch ( changed in another branch ). Dockerfile is not there ( it is lib repo )